docs: add contributing guidelines (#14)
orelbn/tbd: CONTRIBUTING.md

> # Contributing

Reviewer: ```suggestion
# Contributing 💖
```

> Thank you for your interest in contributing to this project! We welcome contributions of all kinds, including bug reports, feature requests, and design ideas. We are at an early stage of the development process so breaking changes may occur often. We appreciate your patience and understanding.

Reviewer: ```suggestion
Welcome to the repo and thank you for your interest in contributing to this project! We welcome contributions of all kinds, including bug reports, feature requests, and design ideas. We are at an early stage of the development process so breaking changes may occur often. We appreciate your patience and understanding.
```

## How to Contribute

- **Open Issues:**  
  Please open a new issue if you encounter any bugs or have feature suggestions. Your feedback helps improve the project.

- **Design Contributions:**  

>   We are in an early stage of development and welcome individuals with design ideas. If you have a proposal or design improvements, please feel free to add your ideas under the design folder.

Reviewer: ```suggestion
  We are in an early stage of development and welcome any new design ideas. If you have a proposal or design improvements, please open a discussion to engage the maintainers and the community.
```

- **Pull Requests:**  

>   We welcome pull requests for bug fixes, new features, and design improvements. Pull requests templates should be available shortly. For now please provide a clear description of the changes made. The testing that was conducted, and any relevant information that would help the reviewer understand the changes. Please also highlight any breaking changes that may have been introduced.

Reviewer: ```suggestion
  We welcome pull requests for bug fixes, new features, and design improvements. Ideally, PRs are created from existing issues or discussions so that the community has context over the changes you are proposing. Pull requests templates should be available shortly. For now please provide a clear description of the changes made. The testing that was conducted, and any relevant information that would help the reviewer understand the changes. Please also highlight any breaking changes that may have been introduced.
```
